packages/appium/lib/utils.js

```diff
@@ -1,4 +1,3 @@
-// @ts-check
```

Comment:
what is the consequence of removing `// @ts-check` from files?

Reply:
`// @ts-check` tells typescript "check this file" regardless if it is configured to check the file.  previously, typescript was _not_ configured to typecheck _any_ javascript, and we enabled it on a per-file basis instead.  now that is no longer true--at least for the `appium` package--as we've added `checkJs: true` to its `tsconfig.json`.  thus the `// @ts-check`s are redundant.